indico/modules/auth/controllers.py

```diff
@@ -108,13 +108,14 @@ def _process(self, force):
         if request.method == 'POST':
             active_provider = provider = _get_provider(request.form['_provider'], False)
             form = provider.login_form()
-            rate_limit_exceeded = not login_rate_limiter.test()
-            if not rate_limit_exceeded and form.validate_on_submit():
-                response = multipass.handle_login_form(provider, form.data)
-                if response:
-                    return response
-                # re-check since a failed login may have triggered the rate limit
-                rate_limit_exceeded = not login_rate_limiter.test()
+            if form.validate_on_submit():
+                rate_limit_exceeded = not login_rate_limiter.test(form.data['identifier'])
```

Comment:
```suggestion
                rate_limit_exceeded = not login_rate_limiter.test(form.identifier.data)
```

Otherwise it unnecessarily builds a dict just to get a single value from it